base/file.jl

```diff
@@ -341,37 +353,47 @@ Obtain the path of a temporary directory (possibly shared with other processes).
 tempdir()
 
 """
-    tempname()
+    tempname(;prefix="jl_")
+
+Generate a unique temporary file path. A `prefix` can be provided in addition, which
+otherwise defaults to `jl_`.
```

Comment:
No need to repeat the default. OTOH, you can mention the truncation directly in that sentence.